understanding/22/consistent-help.html

```diff
@@ -67,40 +90,20 @@ <h2>Intent of Consistent Help</h2>
          </ul>
          <p>While it would be best for all sites to offer at least one of these types of help, this SC does not require help be provided. Also, the order of the types of help listed in the SC does not imply priority. The SC requires that when one of the forms of help listed is available that it be in a consistent location.</p>
```

Comment:
following on from my previous comment about moving the sense of this directly into the intent section, this could then be slimmed down to just

`The order of the types of help listed in the SC does not imply priority.`